cmd/keycloakb/keycloak_bridge.go

```diff
@@ -215,6 +215,13 @@ func main() {
 		}
 	}
 
+	// Security - allowed trustID groups
+	var trustIDGroups = strings.Split(c.GetString("trustid-groups"), ",")
```

Comment:
You can use c.GetStringSlice("trustid-groups") and have them as multiple values in the config file (see cors config)

Reply:
fixed